Content.Shared/Silicons/Laws/Components/SiliconLawProviderComponent.cs

```diff
@@ -1,12 +1,13 @@
-﻿using Robust.Shared.Prototypes;
+﻿using Content.Shared._DV.Silicons; // DeltaV - LawElectronicsInsertionSystem
+using Robust.Shared.Prototypes;
 using Robust.Shared.Audio;
 
 namespace Content.Shared.Silicons.Laws.Components;
 
 /// <summary>
 /// This is used for an entity which grants laws to a <see cref="SiliconLawBoundComponent"/>
 /// </summary>
-[RegisterComponent, Access(typeof(SharedSiliconLawSystem))]
+[RegisterComponent, Access(typeof(SharedSiliconLawSystem), typeof(LawElectronicsInsertionSystem))] // DeltaV - we need this other system to mutate laws too
```

Comment:
should probably specify what "this other system" is in the comment
```suggestion
[RegisterComponent, Access(typeof(SharedSiliconLawSystem), typeof(LawElectronicsInsertionSystem))] // DeltaV - we need LawElectronicsInsertionSystem to mutate laws too
```